clang/include/clang/Driver/Action.h

```diff
@@ -91,7 +91,12 @@ class Action {
     OFK_Cuda = 0x02,
     OFK_OpenMP = 0x04,
     OFK_HIP = 0x08,
-    OFK_SYCL = 0x10
+    OFK_SYCL = 0x10,
+
```

Comment:
Is it possible that all the FPGA use cases are under OFK_SYCL, and this enum is not changed? FPGA does not really define a new offload kind conceptually - it is still SYCL.